bot/src/application/chart/ChartRenderer.ts

```
@@ -14,54 +42,11 @@ export class ChartRenderer {
-            "Error in advanced chart rendering, falling back to basic",
-            error
-          );
-          // Fall back to basic rendering
-        }
-      }
-
-      // Basic mock implementation
-      logger.info("Using basic chart rendering (feature flag disabled)");
-      const mockChartBuffer = Buffer.from("Basic chart rendering output");
-      return mockChartBuffer;
+      const strategy = this.getStrategy();
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)